src/Data/Managers/V2/helpers/useQueries.ts

```diff
@@ -79,6 +79,83 @@ export const useGetWithoutEnv = (options?: { message?: string }) => {
   };
 };
 
+/**
+ * Custom hook to perform a GET Zip request.
+ *
+ * @param {Object} [options] - Optional configuration for the GET request.
+ * @param {string} [options.message] - Optional message to include in the request headers.
+ *
+ * This hook constructs the base URL and headers, and provides a function to perform a GET request to the specified path.
+ *
+ * @returns {Function} A function that performs a GET zip request and returns the response blob.
+ *
+ * @template- The type of the response data.
```

Comment:
```suggestion
 * @template T - The type of the response data.
```
Just learning about template typing. If you want to use it, this is the correct way of declaring it in the docstrings. And I would also expect the typing to be present in the typescript code in that case.

```TS
export const useGetZip = <T>(options?: { message?: string }): (path: string) => Promise<T> => {...}
```

And then it would technically allow you to do the following when using the function:

```TS
const getZip = useGetZip<{ data: string }>({ message: "Hello" });
getZip("/path/to/zip").then(response => {
  console.log(response.data);
});
```

I can see the value of using such a thing, especially with some queries that you'd want to make generic. But perhaps this is too deep into the typing system for us at the moment.